### ISSUE TITLE
Large loss with slow descent

### ISSUE THREAD

**Issue by cwxcode** [Belval/CRNN#20]
Hi Belval ! When I use your code to train with my training data of 100,000 pictures, I got a large loss and the loss descend slowly as follows:

![problem](https://user-images.githubusercontent.com/33143843/42263328-474ee836-7fa0-11e8-8bb1-6a33cfc612bd.PNG)

How can I solve this problem?
Thank you!

**Comment by Belval** [comment]
Unfortunately I don't have much advice to offer on this because `ctc_loss` is mostly a blackbox for me. Some training I did had high loss yet performed a lot better than other lower-loss training.
